snow/snowtest/snowtest.go

```diff
@@ -28,3 +40,39 @@ func ConsensusContext() *snow.ConsensusContext {
 		VertexAcceptor:      noOpAcceptor{},
 	}
 }
+
+func Context(tb testing.TB) *snow.Context {
+	require := require.New(tb)
+
+	ctx := snow.DefaultContextTest()
+
+	ctx.NetworkID = constants.UnitTestID
+	ctx.SubnetID = constants.PrimaryNetworkID
+	ctx.XChainID = ids.GenerateTestID()
+	ctx.CChainID = ids.GenerateTestID()
+	ctx.AVAXAssetID = ids.GenerateTestID()
```

Comment:
This is a slight change wrt context in avm. That context had these values hardcoded; this context has values changing from test to test and depending on the order of execution.
It should not really matter, just pointing it out

Reply:
I changed these to be random since we shouldn't be relying on these being constant anyway. The only one that's guaranteed to always be the same is the PlatformChainID